include/nighthawk/common/version.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "absl/strings/str_cat.h"
+
+namespace Nighthawk {
+
+const int MAJOR_VERSION{0};
+const int MINOR_VERSION{3};
```

Comment:
Do you want full semantic major.minor.patch style versions?

Reply:
Lets go for that, I'll amend